### ISSUE TITLE
Change in attached console output behavior? (no output in compose plugin?) 

### ISSUE THREAD

**Issue by scyto** [docker/compose#9283]
**Description**

**Steps to reproduce the issue:**
1. use the compose provided lower down in this issue
2. try `docker compose up` vs `docker-compose up`
3. see difference 

**Describe the results you received:**

![image](https://user-images.githubusercontent.com/11369180/158218302-6352e37e-d1ed-402e-89de-111d72ca9e55.png)

**Describe the results you expected:**
![image](https://user-images.githubusercontent.com/11369180/158219503-9c328941-c252-4eee-81cc-87d013a6d13a.png)

**Additional information you deem important (e.g. issue happens only occasionally):**
this is the compose file
```
version: "3.7"
services:
  zwavejs2mqtt:
    container_name: zwavejs2mqtt
    image: zwavejs/zwavejs2mqtt:latest
    restart: always
    tty: true
    stop_signal: SIGINT
    stop_grace_period: 10s
    networks:
      - zwave
    devices:
      - "/dev/serial/by-id/usb-Silicon_Labs_CP2102N_USB_to_UART_Bridge_Controller_7eabf24edc08ea118466d2352473482f-if00-port0:/dev/ttyUSB0"
    volumes:
      - ./store:/usr/src/app/store
      - /etc/timezone:/etc/timezone:ro
      - /etc/localtime:/etc/localtime:ro
    environment:
      - TZ=America/Los_Angeles
    ports:
      - "80:8091"
      - "3000:3000" 

networks:
  zwave:
```

**Output of `docker compose version`:**

```
Docker Compose version v2.3.3
```

**Output of `docker info`:**

```
pi@pi-zwave01:~/zwavejs2mqtt $ docker info
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Build with BuildKit (Docker Inc., v0.6.1-docker)
  compose: Docker Compose (Docker Inc., v2.3.3)

Server:
 Containers: 2
  Running: 1
  Paused: 0
  Stopped: 1
 Images: 26
 Server Version: 20.10.13
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: e25210fe30a0a703442421b0f60afac609f950a3
 runc version: v1.0.1-0-g4144b63
 init version: de40ad0
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 5.10.52-v7l+
 Operating System: Raspbian GNU/Linux 10 (buster)
 OSType: linux
 Architecture: armv7l
 CPUs: 4
 Total Memory: 7.762GiB
 Name: pi-zwave01
 ID: 7JXJ:STOV:OEKX:SSH4:3QUF:ZIMG:5W56:UWAW:NTM2:GRVM:BUGV:4JYV
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No memory limit support
WARNING: No swap limit support
WARNING: No kernel memory TCP limit support
WARNING: No oom kill disable support
```

**Additional environment details:**
raspberrypi

**Comment by Anexen** [comment]
Probably related to https://github.com/docker/compose/issues/9288

**Comment by scyto** [comment]
> Probably related to #9288

aye removing tty:true fixed my symptoms, closing this issue as #9288 better documents the change in behaviour.
